### PR TITLE
Iosxe/show cts rbacl

### PR DESCRIPTION
**Pull request by reustaquio22**
Parser for Iosxe/show cts rbacl

### CODE REVIEW

File: src/genie/libs/parser/iosxe/tests/test_show_cts.py [hunk 1]
```diff
@@ -2,9 +2,121 @@
...
+# =================================
+class TestShowCtsRbacl(unittest.TestCase):
+    """Unit test for 'show cts rbacl'"""
+
+    maxDiff = None
+    empty_output = {'execute.return_value': ''}
+    golden_parsed_output1 = {
+    "cts_rbacl": {
+        "ip_ver_support": "IPv4 & IPv6",
+        "TCP_51005-01": {
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/tests/test_show_cts.py:19]: Let's have a key 'name' for these.

```
{
    "cts_rbacl": {
        "ip_ver_support": "IPv4 & IPv6",
        "name": {                          # ADDED
            "TCP_51005-01": {
                "ip_protocol_version": "IPV4",
                (snip)

File: src/genie/libs/parser/iosxe/tests/test_show_cts.py [hunk 2]
```diff
@@ -2,9 +2,121 @@
...
+            "ip_protocol_version": "IPV4",
+            "refcnt": 2,
+            "flag": "0x41000000",
+            "stale": False,
+            1: {
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/tests/test_show_cts.py:24]: let's have a key `aces` for ACEs.
```
'aces': { # ADDED
    1: {
        "action": "permit",
        (snip)

File: src/genie/libs/parser/iosxe/tests/test_show_cts.py (hunk 1 above)
Comment (reustaquio22) [reply src/genie/libs/parser/iosxe/tests/test_show_cts.py]: This has been updated.

File: src/genie/libs/parser/iosxe/tests/test_show_cts.py (hunk 2 above)
Comment (reustaquio22) [reply src/genie/libs/parser/iosxe/tests/test_show_cts.py]: This has been updated.

File: src/genie/libs/parser/iosxe/tests/test_show_cts.py (hunk 1 above)
Comment (tahigash) [reply src/genie/libs/parser/iosxe/tests/test_show_cts.py]: confirmed

File: src/genie/libs/parser/iosxe/tests/test_show_cts.py (hunk 2 above)
Comment (tahigash) [reply src/genie/libs/parser/iosxe/tests/test_show_cts.py]: confirmed

**Comment by ThomasJRyan** [comment]
Please resolve merge conflicts before we can merge @reustaquio22

**Comment by reustaquio22** [comment]
@ThomasJRyan, Conflicts have been resolved

**Comment by ThomasJRyan** [comment]
@reustaquio22 I still seem to be seeing conflicts, specifically with `changelog/undistributed.rst`. We've actually changed the procedure for changelogs which I overlooked on my first review.

Can you please put your changelog into the `changelog/undistributed/` folder following the template format? That should resolve all issues and allow us to merge. Thank you!

**Comment by jeaubin** [comment]
@reustaquio22  - if you get the conflict resolved - we can merge this today

**Comment by itdependsnetworks** [comment]
please close, covered in https://github.com/CiscoTestAutomation/genieparser/pull/250

**Comment by tahigash** [comment]
as per request, closed.

**Pull request closed without merging**